File: server/events/pkg/types/event.go
package types

import (
	"time"
)

type Event struct {
	ID      		string   	`json:"id"`
	Title    		string 		`json:"title"`
	Info    		string 		`json:"info"`
	CreatedAt 		time.Time	`json:"created"`
	UpdatedAt 		time.Time	`json:"updated"`
}

// BeforeInsert set CreatedAt and UpdatedAt.
func (e *Event) BeforeInsert() error {
	e.CreatedAt = time.Now().UTC().Truncate(time.Second)
	e.UpdatedAt = e.CreatedAt
	return nil
}

// BeforeUpdate set UpdatedAt.
func (e *Event) BeforeUpdate() error {
	e.UpdatedAt = time.Now().UTC().Truncate(time.Second)
	return nil
}

File: serverless/src/api/resolvers/event.ts
export default {
    Event: {
        id: event => event.id,
        title: event => event.title,
        description: event => event.description,
        location: event => event.location,
        createdAt: event => event.createdAt,
        members: event => event.members,
        organizers: event => event.organizers
    },
};